sdk/cosmos/azure-cosmos/src/test/java/com/azure/cosmos/rx/ContainerQueryTest.java

```diff
@@ -109,8 +110,41 @@ public void queryCollections_NoResults() throws Exception {
         validateQuerySuccess(queryObservable.byPage(), validator);
     }
 
+    @Test(groups = { "simple" }, timeOut = TIMEOUT)
```

Comment:
this test group will run against prod endpoint.

Creating container/deleting container is expensive against prod. We usually use emulator for such tests.
Please see this test class: CollectionCrudTest.java (it uses emulator for test group).

Could you please create a new Java Test class (ContainerCreateDeleteWithSameNameTest.java) similar to CollectionCrudTest and move this test method there?

we need to test more scenarios (point read, write, query, change feed) on container create delete so it is better if we have a separate test class for that.